### DISCUSSION TITLE
403 against github/copilot-preview

### DISCUSSION THREAD

**Discussion by felipecrs** [lowlighter/metrics#587] [Q&A]
I'm receiving 403 error because it tries to fetch some data from `github/copilot-preview`, even though I've added it in the repositories list to ignore.

See this run example:

https://github.com/felipecrs/felipecrs/runs/3981160605?check_suite_focus=true#step:2:840

I'm not a collaborator of `github/copilot-preview`, I've just joined its preview and I got invited to it because of that (as other thousands of people).

**Comment by lowlighter** [comment]
Will be tracked in #597 👍

**Reply by lowlighter** [reply]
Should be fixed with #678
